MobileWallet/Common/Theme.swift

```diff
@@ -116,6 +119,9 @@ struct Fonts: Loopable {
     let splashTestnetFooterLabel = UIFont(name: "AvenirLTStd-Heavy", size: 9.0)
     let actionButton = UIFont(name: "AvenirLTStd-Heavy", size: 16.0)
 
+    //Splash
+    let splashTitleLabel = UIFont(name: "AvenirLTStd-Black", size: 30.0)
```

Comment:
Add these as a property inside `Theme.swift`. Any references to fonts, colors, images we're keeping in a theme file

Reply:
Sorry about that I didn't see what file this was added to 🤦‍♂